service/controller/key/cluster.go

```diff
@@ -31,14 +31,16 @@ const (
 
 // AWS Tags used for cost analysis and general resource tagging.
 const (
-	TagAvailabilityZone  = "giantswarm.io/availability-zone"
-	TagCluster           = "giantswarm.io/cluster"
-	TagInstallation      = "giantswarm.io/installation"
-	TagMachineDeployment = "giantswarm.io/machine-deployment"
-	TagOrganization      = "giantswarm.io/organization"
-	TagRouteTableType    = "giantswarm.io/route-table-type"
-	TagStack             = "giantswarm.io/stack"
-	TagSubnetType        = "giantswarm.io/subnet-type"
+	TagAvailabilityZone        = "giantswarm.io/availability-zone"
+	TagCluster                 = "giantswarm.io/cluster"
+	TagClusterType             = "giantswarm.io/cluster-type"
+	TagClusterTypeControlPlane = "control-plane"
```

Comment:
This should have the same prefix/format as the others. Customers work in their AWS Accounts as well and we just increase the likelihood of conflict with these types of labels. The convention so far was to go with the k8s style labelling and we should stick to that IMO.